modules/ztp-image-based-upgrade-prep-oadp.adoc

```diff
@@ -72,65 +74,24 @@ The same version of the applications must function on both the current and the t
 ====
 --
 
-. Create the `oadp-cm` `ConfigMap` object through the `oadp-cm-policy` in a new `PolicyGenTemplate` called `ibu-upgrade-ranGen.yaml`:
-+
-[source,yaml]
-----
-apiVersion: ran.openshift.io/v1
-kind: PolicyGenTemplate
-metadata:
-  name: example-group-ibu
-  namespace: "ztp-group"
-spec:
-  bindingRules:
-    group-du-sno: ""
-  mcp: "master"
-  evaluationInterval: 
-    compliant: 10s
-    noncompliant: 10s
-  sourceFiles:
-  - fileName: ConfigMapGeneric.yaml
-    complianceType: mustonlyhave
-    policyName: "oadp-cm-policy"
-    metadata:
-      name: oadp-cm
-      namespace: openshift-adp
-----
-
 . Create a `kustomization.yaml` with the following content:
 +
 [source,yaml]
 ----
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
-generators: <1>
-- ibu-upgrade-ranGen.yaml
-
-configMapGenerator: <2>
+configMapGenerator: <1>
 - files:
   - source-crs/ibu/PlatformBackupRestore.yaml
```

Comment:
We added another version of this file for IBGU. Use `PlatformBackupRestoreWithIBGU.yaml` in the example and add a note for the case where IBU is performed on spokes directly, for which the original should be used.

Reply:
@jc-rh This was only added to the repository yesterday, right? I checked if there was an IBGU-specific Backup CR but I couldn't find it.

Please note that any further comments will be added post GA.